frame/treasury/src/lib.rs

```diff
@@ -289,6 +289,8 @@ pub mod pallet {
 		InvalidIndex,
 		/// Too many approvals in the queue.
 		TooManyApprovals,
+		/// Proposal not in approval queue.
```

Comment:
"Approval queue" suggests that it's waiting to be approved, but in this case it's looking for proposals that have already been approved.
```suggestion
		/// Proposal has not been approved.
```